utilities/BuildUtilities.groovy

```diff
@@ -822,6 +822,46 @@ def matches(String file, List<PathMatcher> pathMatchers) {
 	return result
 }
 
+/**
+ * generates the IdentifyStatement for the Binder
+ * 
+ * parameter:
+ *  buildFile
+ * 
+ * returns:  
+ *  - IDENTIFY string following the pattern:
+ *    <application>/<abbreviatedGitHash>
+ *  
+ *  - null if statement cannot be generated 
+ *   
+ * additional information:
+ *  https://www.ibm.com/docs/en/zos/2.5.0?topic=reference-identify-statement
+ * 
+ */
+def generateIdentifyStatement(String buildFile) {
+
+	def String identifyStmt
+
+	if((props.mergeBuild || props.impactBuild || props.fullBuild) && MetadataStoreFactory.getMetadataStore() != null) {
+		
+		String member = CopyToPDS.createMemberName(buildFile)
+		String identifyString = props.application + "/" + getShortGitHash(buildFile)
+		//   IDENTIFY EPSCSMRT('MortgageApplication/abcabcabc')
+		identifyStmt = "  " + "IDENTIFY ${member}(\'$identifyString\')"
```

Comment:
It is generally a good idea to also include some *eyecatcher* text that makes this information easier to find when browsing the load module (in contrast to using `amblist`):
`identifyStmt = "  " + "IDENTIFY ${member}(git-hash: \'$identifyString\')"`

That way you can search for git-hash and will always have the information visible even when browsing the load module.

Reply:
Good point, I concur to @Crosswind 's point :)